bin/install_cmdstan

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
+"""
+Download and install a CmdStan release from GitHub.
+Optional command line arguments:
+   -v, --version : version, defaults to latest
+   -d, --dir : install directory, defaults to '~/.cmdstanpy
+"""
+import argparse
+import contextlib
+import os
+import os.path
+import re
+import shutil
+import subprocess
+import sys
+import tarfile
+import urllib.request
```

Comment:
Is there a difference with `request`? Is urllib the default? (vs urllib2, urllib3)

Reply:
for me (Mac OS, Python 3.7.0) there is a difference w/ request - import of `urllib` gives the following error:

> AttributeError: module 'urllib' has no attribute 'request'

Reply:
Yes, the import format is correct

I was thinking https://2.python-requests.org/en/master/

But I think urllib is the correct choice here.